### ISSUE TITLE
Feature request: support for current Android push notification icon scheme

### ISSUE THREAD

**Issue by martindederer** [pushy-me/pushy-cordova#6]
I was wondering if there are any plans to support the current Android push notification icon scheme. If i understood it correctly this API change was introduced in [Android API Version 21 (Lollipop 5.0)](https://developer.android.com/about/versions/android-5.0-changes).

The current implementation of the plugin uses the app icon as push notification icon even when targeting Android API versions >= 21 which is most likely not the intended behavior since only the icon's alpha channel is used for rendering. So depending on the app icon in use either a white square or an outline of your app icon is shown in the status bar/the push notification preview.

It would be nice to be able to define the Android push notification icon (and background color for that matter) at initialization time of the plugin.

https://developer.android.com/guide/practices/ui_guidelines/icon_design_status_bar
https://clevertap.com/blog/fixing-notification-icon-for-android-lollipop-and-above/

**Comment by pushy** [comment]
@martindederer Excellent suggestion!

We've just added this in commit https://github.com/pushy-me/pushy-cordova/commit/2156cab19cb38780eca9678da0605c62c04507db and published to `npm` in version `1.0.20`.

To start using this feature:

```
cordova plugin remove pushy-cordova-receiver
cordova plugin remove pushy-cordova

cordova plugin add pushy-cordova@1.0.20
cordova plugin add ./plugins/pushy-cordova/receiver/
```

Then, add the relevant icon image file(s) (e.g. `ic_notification.png`) in the following path:
```
platforms/android/app/src/main/res/drawable-*
```

Finally, in your `www/js/index.js`, find:

```
Pushy.listen();
```

Add below:
```
Pushy.setNotificationIcon('ic_notification');
```

**Comment by martindederer** [comment]
Very nice! I'll give it a try.
